### DISCUSSION TITLE
BIQU microprobe support

### DISCUSSION THREAD

**Discussion by 3dYourLife** [mriscoc/Ender3V2S1#1303] [Q&A]
I just got a BIQU V2 microprobe and trying to build/ find a firmware that will work with the stock ender 3 v2 4.2.2 board. I tried to download a stock marlin build and edit the config files as seen in another post but when I did it would either come back with error or when i pushed the bin folder to the printer it would just turn the screen red after flashing the firmware. I'm a super newb with coding and need any and all help i can get.....

**Comment by mriscoc** [comment]
Comments about the use of the BIQU Microprobe: https://github.com/bigtreetech/MicroProbe/issues/14#issuecomment-1822195498

To compile firmware compatible with the BIQU Microprobe, you must select the levelling device in the Configurator:

![image](https://github.com/user-attachments/assets/d4a5349b-c145-42df-af67-01d95e65ceae)

For the SKR Mini boards, use the cable adapter to connect the black and white wires to the Z-end stop connector of the Mini board, as recomendad in the Bq MP Manual:
  
![image](https://github.com/user-attachments/assets/8b4cfebd-9637-4a5c-84f7-e1617419140d)

And the other tree cables to bltouch port, the operation of the Biqu microprobe will be not reliable if its output are connected to the bltouch port of the Mini board, additionally verify `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN` and `#define PROBE_ENABLE_PIN  PA1`, they should be enabled.
